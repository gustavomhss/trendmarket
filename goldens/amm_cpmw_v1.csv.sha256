3434cdf21e2f165d4da55e70047aac5d1af71e6453567bb5c53b93844a7b9e2a  goldens/amm_cpmw_v1.csv
