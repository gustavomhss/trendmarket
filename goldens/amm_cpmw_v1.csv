id,op,x_wad,y_wad,dx_wad,dy_wad,fee_ppm,expect_kind,expect_wad
OUT_SYM_NOFEE,OUT,1000000000000000000000000,1000000000000000000000000,10000000000000000000000,0,0,ok,9900990099009900990099
OUT_SYM_FEE,OUT,1000000000000000000000000,1000000000000000000000000,10000000000000000000000,0,3000,ok,9871580343970612988505
OUT_ASYM_NOFEE,OUT,1000000000000000000000,1000000000000000000000000000,100000000000000000000,0,0,ok,90909090909090909090909091
OUT_FEE_DX1_TOO_SMALL,OUT,5000000000000000000000000,4000000000000000000000000,1,0,3000,err:InputTooSmall,
OUT_MIN_RESERVE_GUARD,OUT,1000000000000000000000000,1000000000000000005,10000000000000000000000,0,0,err:MinReserveBreached,
IN_SYM_FEE,IN,1000000000000000000000000,1000000000000000000000000,0,9870000000000000000000,3000,ok,9998383138872547672408
IN_SYM_NOFEE,IN,1000000000000000000000000,1000000000000000000000000,0,9900000000000000000000,0,ok,9998990001009998990001
IN_MIN_RESERVE_BREACH,IN,1000000000000000000000000,1000000000000000000000000,0,999999000000000000000000,0,err:MinReserveBreached,
IN_ASYM_FEE,IN,1000000000000000000000,1000000000000000000000000000,0,90000000000000000000,3000,ok,90270820561687
